File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.py
word = input().strip()
croatia_words = ["c=","c-","dz=","d-","lj","nj","s=","z="]

croatia_words.sort(key=len, reverse=True)

for char in croatia_words:
    word = word.replace(char, "*")

print(len(word))

File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin").toString().trim()

const word_arr = ["c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="]
word_arr.sort((a, b) => a.length - b.length).reverse()
let joinInput = input

for(let word of word_arr) {
    joinInput = joinInput.replaceAll(word, "*")
}

console.log(joinInput.length)